sdk/aqueduct/artifacts/bool_artifact.py

```diff
@@ -22,23 +23,37 @@ class CheckArtifact(Artifact):
     Examples:
         >>> @check
         >>> def check_something(df1, metric) -> bool:
-        >>> return check_result
+        >>>     return check_result
         >>>
         >>> check_artifact = check_something(table_artifact, metric_artifact)
 
-        The contents of the check artifact can be manifested locally:
+        The contents of the bool artifact can be manifested locally:
 
         >>> assert check_artifact.get()
     """
 
-    def __init__(self, dag: DAG, artifact_id: uuid.UUID, from_flow_run: bool = False):
+    def __init__(
```

Comment:
Check that we can't consume the output of @checks and @metrics? I think we removed those guardrails on the backend